geomesa-blobstore/README.md

```diff
@@ -1,3 +1,116 @@
 # GeoMesa Blob Store
 
 See [GeoMesa Blob Store](../docs/user/blobstore.rst) in the [GeoMesa User Manual](http://geomesa.org/documentation).
+=======
+#GeoMesa BlobStore
```

Comment:
this is the result from the rebase on master and changes to the documentation @mdzimmerman was your intent that we would no longer use/update the README files?